README.md

```diff
@@ -26,24 +27,53 @@ The main OpenStack entry point is at http://www.openstack.org
  * v2.0:	2017-09-25 - More focus on the DevOps life-cycle
  * v2.1:	2017-10-20 - Review to match 2017 expectations
 
-# Customer Story
+# Project Story
+
+You are in a company which achieved an aggressive deal with a Customer
+to leverage an open source cloud native application and use it internally.
+
+With a new team, you are in charge to deliver and maintain
+the application in production in your new customer environment.
+
+The customer gave you 4 major objectives:
+ - The application is an open source cloud native application:
+    lottery for an e-commerce site
+ - The customer requires the production environment to be Openstack cloud
+     The customer won't install or maintain it. This will one of your task.
+ - Any feature/bug request must be in production quickly, with control &
+    quality.
+ - You will have 7 weeks to deliver the application on the customer Hardware.
+   Before that date, the environment won't be considered as production
+   and can be re-build from scratch, anytime. Just contact the customer
+   to do it.
+
+The Customer will give you 2 servers pre-installed with CentOS 7, to deliver
+the application in production.
+
+Your company gives you an access to an openstack tenant to develop your
```

Comment:
to a reference OpenStack setup hosting a tenant ...

Reply:
fixed